### ISSUE TITLE
复现

### ISSUE THREAD

**Issue by Gravity-vV** [umd-huang-lab/VLM-Poisoning#4]
请问如果只有40G的显存的显卡，有没有办法跑这个训练呢

**Comment by Yuancheng-Xu** [comment]
I think so, just use batch_size_per_device=1 and a higher gradient accumulation value.

**Comment by mattxu98** [comment]
On an RTX 4090 (24G), a line of `bash poison_llava.sh --batch_size 512 --task_name Biden_base_Trump_target`  has been running for more than 24 hours and for at least the next 12 hours

**Comment by Yuancheng-Xu** [comment]
@mattxu98 I think the original post is asking about training the llava model. `poison_llava.sh` is for crafting the poison images. 

I am surprised that 24G can handle a 512 batch size. Could you try to use a smaller batch_size (such as 10) and see whether it is still too slow?
